tests/functional/test_wheel.py

```diff
@@ -289,6 +289,17 @@ def test_pip_wheel_with_user_set_in_config(script, data, common_wheels):
     assert "Successfully built withpyproject" in result.stdout, result.stdout
 
 
+def test_pip_wheel_ext_module_with_tmpdir_inside(script, data, common_wheels):
```

Comment:
The test might fail without a C compiler. I'm running it now in such environment to make sure. Do we have a decorator to skip if a compiler does not exists?

Reply:
I wanted to skip the test based on stderr output, but it seems that the `script` fixture only allows me to either expect success or expect failure and it doesn't allow me to allow both and check manually :(

Reply:
I could do a trick to not depend on gcc:

```python
script.environ['CC'] = str('true')
script.environ['LDSHARED'] = str('true')
```

But not sure how/if that works on Windows.

Will try using `python -c ''` as a substitute.

Reply:
I've pushed code that works on Linux. Hopefully it will also work on Windows.

Reply:
So the environment variables are completely ignored on Windows. A fix might be to actually make extension.c a valid extension module :(